src/pages/Profile/Profile.vue

```diff
@@ -1,13 +1,110 @@
 <template>
-  <div>Profile</div>
+ 
+    <v-flex class="white elevation-2 fluid">
+        <v-toolbar flat dense class="blue darken-3" dark>
```

Comment:
Can you remove toolbar ?
It's for being similar as other pages

For settings we can group using this, but in here it seems unnecessary